kfctl/README.md

```diff
@@ -0,0 +1,140 @@
+# kfctl golang client
+
+## Overview
+
+The kfctl golang client will provide a CLI similar to kfctl.sh but will 
+address the following deficits in the current tool
+
+1. No longer require a local install of ks (ksonnet)
```

Comment:
Can you explain this goal? Do you intend to duplicate the functionality of ks? 
The intent with kfctl.sh was that if you wanted to customize individual components you would have to go into the ksonnet app directory?